data/json/recipes/ammo/shot.json

```diff
@@ -43,10 +43,7 @@
     "charges": 1,
     "using": [ [ "ammo_shot", 1 ] ],
     "tools": [ [ [ "press", -1 ] ] ],
-    "components": [
-      [ [ "gunpowder", 6 ], [ "chem_black_powder", 6 ] ],
-      [ [ "combatnail", 10 ] ]
-    ]
+    "components": [ [ [ "gunpowder", 6 ], [ "chem_black_powder", 6 ] ], [ [ "combatnail", 10 ] ] ]
```

Comment:
This change is questionable, might need to do something about multiply-nested arrays.

Reply:
Then entries in main array should wrap if:
* Nested arrays wrap
* There are size > 1 nested arrays

The main problem here is the number of `[]`. When all nested arrays are size 1, this is not a problem because the meaning is still obvious from context.

Reply:
As with https://github.com/CleverRaven/Cataclysm-DDA/pull/21612/files#diff-b845aefebd7df667ac2f080c259bfc5eR23
the innermost element should be an object, yielding:
```
 "components": [ {
    "gunpowder": 6,
    "chem_black_powder": 6
  }, {
    "combatnail": 10
  }
]
```